lib/src/base-styles/_nx-tile.scss

```diff
@@ -157,8 +157,21 @@ $nx-tile-background-color: $nx-white;
 
   border-top: 1px solid #e9e9e9;
   margin-top: $nx-spacing-md;
-  padding: $nx-spacing-xl 0;
+  padding: $nx-spacing-l 0;
   text-align: right;
```

Comment:
Should be removed now, unless there is an expectation that other footer content aside from a button bar should also be right-aligned.

Reply:
> Should be removed now, unless there is an expectation that other footer content aside from a button bar should also be right-aligned.

https://github.com/sonatype/sonatype-react-shared-components/pull/124/commits/6bc60547bcdc20d4f8eb364a2e584b0f091870e1